docs/dev/cli/saucectl/usage/ci/gitlab.md

```diff
@@ -13,14 +13,15 @@ These examples can apply to virtually any GitLab deployment, provided that you a
 
 ## What You'll Need
 
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
```

Comment:
Should all list item end with a period?

Reply:
Good catch! None of them should because [they're not complete sentences](https://wiki.saucelabs.com/pages/viewpage.action?spaceKey=PM&title=Technical+Documentation+Style+Guide#TechnicalDocumentationStyleGuide-BulletedLists).